Lib/test/mock_socket.py

```diff
@@ -107,16 +107,16 @@ def getpeername(self):
     def close(self):
         pass
 
+    def connect(self, host):
+        pass
+
 
 def socket(family=None, type=None, proto=None):
     return MockSocket(family)
 
+
 def create_connection(address, timeout=socket_module._GLOBAL_DEFAULT_TIMEOUT,
                       source_address=None):
-    try:
-        int_port = int(address[1])
```

Comment:
Please revert this change.
IMHO, this code was intended for checking port value validation.